yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -21,9 +21,11 @@ import java.util.Collections
 import java.util.concurrent._
 import java.util.regex.Pattern
 
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
```

Comment:
use of `JavaConversions` is discouraged -- its very easy to get some implicit conversions happening when you had no intention of converting anything.  stick w/ just `JavaConverters`